src/app/globals.css

```diff
@@ -15,9 +15,9 @@
   }
 }
 
-.swiper-slide {
-  width: auto !important;
-}
+/*.swiper-slide {*/
+/*    width: auto !important;*/
+/*}*/
```

Comment:
if the swiper-slide is unused maybe it's okay to remove it?

Reply:
thanks! Forgot to remove it